src/views/QuizView/quizView.js

```diff
@@ -141,3 +155,21 @@ function previousQuestion() {
   current--;
   renderQuizData();
 }
+
+function useLifeline() {
+  lifeline = true;
+  const lifelineDiv = document.getElementById('lifeline');
+  lifelineDiv.innerHTML = '';
+
+  const answersContainer = document.getElementById('answers');
+  const answers = answersContainer.children;
+  let removed = 0;
+  let n;
+  while (removed < 2) {
+    n = Math.floor(Math.random() * answers.length);
+    if (questions[current].correct.toUpperCase() != answers[n].children[0].innerText) {
+      answersContainer.removeChild(answers[n]);
+      removed++;
+    }
+  }
```

Comment:
With this implementation when user navigates back, all 4 answers are available. Try to remove answers directly from Question object referenced in the Answer object.